conjure-core/src/main/java/com/palantir/conjure/parser/services/PathString.java

```diff
@@ -18,11 +18,13 @@
 
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.palantir.conjure.defs.ConjureImmutablesStyle;
+import com.palantir.logsafe.Unsafe;
 import com.palantir.util.syntacticpath.Path;
 import com.palantir.util.syntacticpath.Paths;
 import org.immutables.value.Value;
 
 /** Represents a HTTP path in a {@link ServiceDefinition conjure service definition}. */
+@Unsafe
```

Comment:
Actually, should the `path()` below be marked as `@Safe` given it's known at compile-time?